sklearn/datasets/descr/lfw.rst

```diff
@@ -114,7 +114,7 @@ Features            real, between 0 and 255
 
 * `Labeled Faces in the Wild: A Database for Studying Face Recognition
   in Unconstrained Environments.
-  <http://vis-www.cs.umass.edu/lfw/lfw.pdf>`_
```

Comment:
This umass link still seems to work: https://people.cs.umass.edu/~elm/papers/lfw.pdf